types/pg/index.d.ts

```diff
@@ -33,8 +34,16 @@ export interface Defaults extends ConnectionConfig {
 
 import { ConnectionOptions } from "tls";
 
+// Note: modes 'disable' and 'allow' don't work correctly. If property `ssl` is not undefined, SSL
```

Comment:
Are they types supported by the library though? I worry about editorializing the library with the type definition.